pytorch/serving/config.properties

```diff
@@ -10,3 +10,5 @@ allowed_urls=https://s3.amazonaws.com/.*,https://torchserve.pytorch.org/.*
 ipex_enable=true
 cpu_launcher_enable=true
 cpu_launcher_args=--use_logical_core
+disable_token_authorization=true
+enable_model_api=true
```

Comment:
Can you add some kind of update to the README that explains what these settings do, and why users should change these before deploying into production?

Reply:
Added in the README